crates/pop-parachains/src/up/mod.rs

```diff
@@ -206,6 +214,16 @@ impl Zombienet {
 				continue;
 			}
 
+			// Check if command references a parachain template binary without a specified path
+			// (e.g. Polkadot SDK parachain template)
+			if command == "parachain-template-node" || command == "substrate-contracts-node" {
+				let binary_path = PathBuf::from("./target/release").join(&command);
```

Comment:
Could this assumption become problematic if the binary is built with a different profile?